tikv-client/src/main/java/com/pingcap/tikv/TiSession.java

```diff
@@ -187,12 +189,17 @@ public void injectCallBackFunc(Function<CacheInvalidateEvent, Void> callBackFunc
   }
 
   public static void clearCache() {
```

Comment:
is it actually here that we lost synchronized?

Reply:
yes